### ISSUE TITLE
Minor indentation issue with error output

### ISSUE THREAD

**Issue by tasomaniac** [fourlastor/pickle#21]
The newly added check for arguments mismatch in #12 has an indentation issue shown below.  It is minor but would be good to fix.

```
error:                 Pickle Error:
                  Step definition argument mismatch.
  > Step definition: "display network connection error message"
  > Step implementation: com.example.login.LoginSteps.displayNetworkConnectionErrorMessage(java.lang.String)
2 errors
```

**Comment by fourlastor** [comment]
@tasomaniac you did fix this in #51, correct?

**Comment by tasomaniac** [comment]
Yeah, I think this is fixed definitely even if it is not part of #51 😊
